agent/src/index.ts

```diff
@@ -773,6 +774,7 @@ export async function createAgent(
             getSecret(character, "SOLANA_ADMIN_PRIVATE_KEY")
                 ? nftGenerationPlugin
                 : null,
+            getSecret(character, "OPENAI_API_KEY") ? aimePlugin : null,
```

Comment:
I don't think this environment variable is right

Reply:
We have changed the environment variable to .env.example, and also have deleted wrong environment variable